front/types/user.ts

```diff
@@ -30,6 +30,16 @@ export type LimitsType = {
   };
 };
 
+export const FREE_BILLING_VALUES = ["free"] as const;
```

Comment:
Awesome, thank you 🙏 

I would slightly challenge the names "_VALUES"  as to me these are not "values" but "types". But really nit and maybe that's just me

Reply:
Done!